kodi_game_scripting/template_processor.py

```diff
@@ -129,6 +129,8 @@ def __getattr__(self, key):
                         os.path.dirname(os.path.join(destination, outfile)))
                     with open(outfile_path, 'w') as outfile_ctx:
                         outfile_ctx.write(content)
+                elif os.path.exists(outfile_path):
+                    os.remove(outfile_path)
```

Comment:
We'll probably need to revert this again after the next update. For some cores the automatic generation of settings doesn't work as the core dynamically figures out settings based on the loaded game. Now this code removes settings and language files for manually created files that should stay.

Reply:
Do you have a list of affected cores?

Reply:
Yes, those cores would have their settings removed now.

```
game.libretro.tgbdual
game.libretro.mame2003
game.libretro.mame2003_plus
game.libretro.fbalpha2012
game.libretro.fbalpha
game.libretro.sameboy
```

I've updated them manually to keep the files. We can now either revert the patch or another idea might be to add a `.keep` file to the templates directory to control files that shouldn't be removed if they're empty.